beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java

```diff
@@ -100,12 +102,13 @@
     }
   }
 
+  @SuppressWarnings("unused")
```

Comment:
unused?

Reply:
Hmm, it was probably bug in Intellij, removed.